homeassistant/components/generic/config_flow.py

```diff
@@ -380,6 +381,21 @@ async def async_step_user_confirm_still(
 
     async def async_step_import(self, import_config: dict[str, Any]) -> FlowResult:
         """Handle config import from yaml."""
+
+        async_create_issue(
+            self.hass,
+            HOMEASSISTANT_DOMAIN,
+            f"deprecated_yaml_{DOMAIN}",
+            breaks_in_ha_version="2024.2.0",
+            is_fixable=False,
+            issue_domain=DOMAIN,
+            severity=IssueSeverity.WARNING,
+            translation_key="deprecated_yaml",
+            translation_placeholders={
+                "domain": DOMAIN,
+                "integration_title": "generic",
```

Comment:
Isn't it clearer to use "Generic IP camera" as title?

Reply:
Done 83cad79